src/devtools/useAtomDevtools.ts

```diff
@@ -28,96 +57,98 @@ export function useAtomDevtools<Value, Result extends void | Promise<void>>(
   const lastValue = useRef(value)
   const isTimeTraveling = useRef(false)
   const devtools = useRef<
-    ReturnType<NonNullable<typeof extension>['connect']> & {
+    ReturnType<
+      NonNullable<typeof window['__REDUX_DEVTOOLS_EXTENSION__']>['connect']
+    > & {
```

Comment:
Oh, I see. This change is necessary because of `| false`. https://github.com/pmndrs/jotai/pull/1076/files#r846042599
Seems fair then.